### PR TITLE
[Sprint 2] Feature cofirm leave ui #2

### PR DESCRIPTION
**Pull request by tranminhtri911**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -82,6 +82,10 @@
     <string name="compensation_time">Compensation time</string>
     <string name="example_time_from">Ex: 2017/05/20 17:30</string>
     <string name="example_time_to">Ex: 2017/05/20 18:30</string>
+    <string name="confirm_request_leave">Confirm request leave</string>
+    <string name="submit">Submit</string>
+    <string name="next">Next</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: WARNING: [UnusedResources] The resource `R.string.next` appears to be unused
 Unused resources make applications larger and slow down builds.

Comment (tranminhtri911) [inline app/src/main/res/values/strings.xml]: cái này pull sau em mới dùng anh

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -82,6 +82,10 @@
     <string name="compensation_time">Compensation time</string>
     <string name="example_time_from">Ex: 2017/05/20 17:30</string>
     <string name="example_time_to">Ex: 2017/05/20 18:30</string>
+    <string name="confirm_request_leave">Confirm request leave</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: WARNING: [UnusedResources] The resource `R.string.confirm_request_leave` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -82,6 +82,10 @@
...
+    <string name="submit">Submit</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: WARNING: [UnusedResources] The resource `R.string.submit` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java [hunk 4]
```diff
@@ -31,4 +48,80 @@ public void onStart() {
...
+    public void setCurrentOffType(String currentOffType) {
+        mCurrentOffType = currentOffType;
+        notifyPropertyChanged(BR.currentOffType);
+    }
+
+    public void setCurrentPositionOffType(int currentPositionOffType) {
+        mCurrentPositionOffType = currentPositionOffType;
+    }
+
+    @IntDef({
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java]: WARNING: Missing a Javadoc comment.

File: app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java [hunk 5]
```diff
@@ -31,4 +48,80 @@ public void onStart() {
...
+            PositionOffType.OFF_HAVE_SALARY_COMPANY_PAY,
+            PositionOffType.OFF_HAVE_SALARY_INSURANCE_COVERAGE, PositionOffType.OFF_NO_SALARY
+    })
+    @interface PositionOffType {
+        int OFF_HAVE_SALARY_COMPANY_PAY = 0;
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java]: ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.

File: app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java [hunk 6]
```diff
@@ -31,4 +48,80 @@ public void onStart() {
...
+        int OFF_HAVE_SALARY_INSURANCE_COVERAGE = 1;
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java]: ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.

File: app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java [hunk 7]
```diff
@@ -31,4 +48,80 @@ public void onStart() {
...
+        int OFF_NO_SALARY = 2;
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java]: ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.

**Pull request merged**